cobigen-cli/cli/src/main/assembly/resources/bin/cg

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
-    java -cp "$LIB_DIR/*" com.devonfw.cobigen.cli.CobiGenCLI -help
+    java -cp "$(printf %s: $LIB_DIR/*.jar)" -help
```

Comment:
I think you deleted too much here. "com.devonfw.cobigen.cli.CobiGenCLI" is still needed

Reply:
Oops, fixed now.